.github/workflows/ci_workflows.yml

```diff
@@ -59,18 +59,19 @@ jobs:
         #       run only with minimal dependencies.
         - name: Python 3.10 with minimal dependencies and full coverage
           linux: py310-test-cov
+          posargs: --memray
 
         # NOTE: In the build below we also check that tests do not open and
         # leave open any files. This has a performance impact on running the
         # tests, hence why it is not enabled by default.
         - name: Python 3.9 with all optional dependencies
           linux: py39-test-alldeps
           toxargs: -v --develop
-          posargs: --open-files --run-slow
+          posargs: --open-files --run-slow --memray
```

Comment:
Do we need memray in so many jobs?

Reply:
I suppose not, but if we want to start decorating tests for memray to enforce memory limits (and maybe the leak checks), this will require `--memray` to give them actual force.